config/locales/en/nsm/disbursements.yml

```diff
@@ -78,8 +94,17 @@ en:
         nsm/disbursements_form:
           attributes:
             base:
+              no_change: The new values cannot be the same as the values submitted by the provider.
             total_cost_without_vat:
```

Comment:
Potentially misleading error message/the error is potentially triggered at the wrong time: 

If I get a claim with disbursements, then adjust a disbursement and save, then go back to the disbursement and not change anything (keeping my original adjustment in) - we still get this same error even though these aren't the same values submitted by provider

Reply:
Ok, the `data_has_changed?` method now checks for changes compared to the original values, not the most recently adjusted values.